integration/chunks_storage_backends_test.go

```diff
@@ -114,3 +131,187 @@ func TestChunksStorageAllIndexBackends(t *testing.T) {
 	assertServiceMetricsPrefixes(t, Ingester, ingester)
 	assertServiceMetricsPrefixes(t, Querier, querier)
 }
+
+func TestSwiftChunkStorage(t *testing.T) {
```

Comment:
Any reason why you didn't add switf to `TestChunksStorageAllIndexBackends`?

Reply:
Yes like I mentioned offline. This test is mostly for index and refactoring it to test multiple object storage was more difficult than creating a new one more focused and simpler.

Reply:
Sure. As already agreed offline, I'm resolving this comment.